src/NATS.Client.JetStream/Internal/ReplyToDateTimeAndSeq.cs

```diff
@@ -0,0 +1,28 @@
+namespace NATS.Client.JetStream.Internal;
+
+internal static class ReplyToDateTimeAndSeq
+{
+    internal static (DateTimeOffset? DateTime, long? Seq) Parse(string? reply)
```

Comment:
This should return a new `MsgMetadata` class with all the fields populated. For example:

```csharp
class NatsJSMsgMetadata
{
  SequencePair Sequence;
  ulong NumDelivered;
  ulong NumPending;
  DateTimeOffset Timestamp;
  string Stream;
  string Consumer;
  string Domain;
}

class NatsJSSequencePair
{
  ulong Stream;
  ulong Consumer;
}
```

See also Go client implementation https://github.com/nats-io/nats.go/blob/main/jetstream/message.go#L64

Reply:
I was only focusing on Timestamp and Stream Sequence, because this is what I need in my current project.
But you're right better make it right ;)